h2o-docs/src/product/data-science/glm.rst

```diff
@@ -769,6 +774,28 @@ Tweedie Dispersion Example
       model._model_json["output"]["dispersion"]
       8.966819788535565
 
+Negative Binomial
+'''''''''''''''''
+
+GLM dispersion estimation using the maximum likelihood method for the negative binomial family is available when you set ``dispersion_parameter_method=“ml”``.
+
+**Note**: Regularization is not supported when you use dispersion parameter estimation with maximum likelihood.
```

Comment:
This note applies even for the Tweedie and other families when the dispersion is estimated using maximum likelihood. So I'd suggest moving it up in the hierarchy.